scio-repl/src/main/scala/com/spotify/scio/repl/ScioILoop.scala

```diff
@@ -227,13 +227,22 @@ class ScioILoop(scioClassLoader: ScioReplClassLoader,
     }
   }
 
+  private def loadIoCommands(): IR.Result = {
+    intp.interpret(
+      """
+        |val _ioCommands = new com.spotify.scio.repl.IoCommands(sc.options)
```

Comment:
So if I would like to use IOCommands with different opts - i have to manually create it myself right? And by default you get the IOCommands from your startup opts?